manufacturing/internals/blower/README.md

```diff
@@ -87,6 +87,8 @@ There are notes on manufacturing gaskets in the [manufacturing methods guide][mm
 
 ### Acrylic plates
```

Comment:
I'd suggest a minor reorganization of this readme. The assembly section is intersperse with part drawings and links and images. Keep them separate and the assembly instruction condensed with resized images alongside the steps for easy reading.